e2e/cypress/integration/enterprise/system_console/limited_console_access_spec.js

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @system_console
+
+import accessRules from '../../../fixtures/system-roles-console-access';
+import disabledTests from '../../../fixtures/console-example-inputs';
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('Limited console access', () => {
+    const roleNames = ['system_manager', 'system_user_manager', 'system_read_only_admin'];
+    const ACCESS_NONE = 'none';
+    const ACCESS_READ_ONLY = 'read';
+    const ACCESS_READ_WRITE = 'read+write';
+    const testUsers = {};
+
+    before(() => {
+        cy.apiRequireLicense();
+
+        Cypress._.forEach(roleNames, (roleName) => {
+            cy.apiCreateUser().then(({user}) => {
+                testUsers[roleName] = user;
+            });
+        });
+    });
+
+    it('MM-T3386 - Verify the Admin Role - System Manager', () => {
+        const role = 'system_manager';
+
+        // # Make the user a System  Manager
+        makeUserASystemRole(role);
+
+        // * Login as the new user and verify the role permissions (ensure they really are a system manager)
+        forEachConsoleSection(role);
+    });
+
+    it('MM-T3387 - Verify the Admin Role - System User Manager', () => {
+        const role = 'system_user_manager';
+
+        // # Make the user a System User Manager
+        makeUserASystemRole(role);
+
+        // * Login as the new user and verify the role permissions (ensure they really are a system user manager)
+        forEachConsoleSection(role);
+    });
+
+    it('MM-T3388 - Verify the Admin Role - System Read Only Admin', () => {
+        const role = 'system_read_only_admin';
+
+        // # Make the user a System Ready Only Manager
+        makeUserASystemRole(role);
+
+        // * Login as the new user and verify the role permissions (ensure they really are a system read only manager)
+        forEachConsoleSection(role);
+    });
+
+    const makeUserASystemRole = (role) => {
+        // # Login as each new role.
+        cy.apiAdminLogin();
+
+        // # Go the system console.
+        cy.visit('/admin_console/user_management/system_roles');
+        cy.contains('System Roles', {timeout: TIMEOUTS.ONE_MIN}).should('exist').and('be.visible');
+
+        // # Click on edit for the role
+        cy.findByTestId(`${role}_edit`).click();
+
+        // # Click Add People button
+        cy.findByText('Add People').click();
```

Comment:
Following is my suggestion I tested  and it's working   
`cy.findByRole('button', { name: /add people/i }).click()`;

Reply:
Done!